appinventor/appengine/war/Ya.css

```diff
@@ -1475,7 +1475,6 @@ input[type="color"],
   display: inline-block;
   height: 20px;
   padding: 2px 6px;
-  margin-bottom: 4px;
```

Comment:
Rather than remove the bottom margin for all inputs (which condenses the Properties panel), I recommend that we create a CSS class for `ode-LabeledTextBox` that removes the margin-bottom for its nested inputs. This will have the same visual effects for the dialog without affecting the properties panel.

Reply:
I agree because it is more specific to the changes from this pull request.

But in fact, there are a lot of asymmetric padding, border and margin. They somehow make some ui mis-aligned.